config/management/operational/src/command.rs

```diff
@@ -1,29 +1,39 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 use crate::TransactionContext;
+use libra_crypto::x25519;
 use libra_management::error::Error;
+use libra_types::validator_config::ValidatorConfig;
 use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 #[structopt(about = "Tool used for Operators")]
 pub enum Command {
+    #[structopt(about = "Gets the validator config")]
+    GetValidatorConfig(crate::validator_config::GetValidatorConfig),
```

Comment:
why?

Reply:
I can actually drop this.

I was fighting a lot with the `test_helper` in smoke test, and I wanted to just prove to myself that something worked.  We can just use the JSON RPC in the tests rather than making a command for this.